checks/storeonce4x_cat_stores

```diff
@@ -33,9 +33,10 @@ def check_storeonce4x_cat_stores(item, params, parsed):
     user_bytes = store["userBytes"]
     num_items = store["numItems"]
 
-    yield from df_check_filesystem_list_coroutine(
-        item, params, [(item, size_available_mb, size_available_mb - size_used_mb, 0)]
-    )
+    if store["sizeOnDiskQuotaEnabled"] is not False:
```

Comment:
Is "sizeOnDiskQuotaEnabled" always there?

Don't compare boolean values to True or False (see PEP8); better and more robust:

if store.get("sizeOnDiskQuotaEnabled", False):
    ...

Reply:
Yes "sizeOnDiskQuotaEnabled" must exists all the time.
I changed the requested line.